login/routers/controllers/user_controllers/AuthentificationService.js

```diff
@@ -0,0 +1,48 @@
+const sequelize = require('sequelize');
+
+const sequelizeOperators = sequelize.Op;
+const bcrypt = require('bcrypt');
+const userModel = require('../../../db/models/user_models/UserModel.js');
+const userService = require('./UserService.js');
+const createUserResponce = require('../../../db/user_db/CreateUserResponce.js');
+
+
+function signUp(registrationRequest, response) {
+  userModel.user.findOne({
```

Comment:
The duplication is the userModel.user.findOne block which I think previously was the UserExists function? Abstract this block into the userService class